ncloud/resource_ncloud_nks_cluster_test.go

```diff
@@ -0,0 +1,247 @@
+package ncloud
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/NaverCloudPlatform/ncloud-sdk-go-v2/services/vnks"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccResourceNcloudNKSCluster_basic(t *testing.T) {
+	var cluster vnks.Cluster
+	name := getTestClusterName()
+	clusterType := "SVR.VNKS.STAND.C002.M008.NET.SSD.B050.G002"
+	k8sVersion := "1.19.14-nks.1"
+	resourceName := "ncloud_nks_cluster.cluster"
+
+	resource.Test(t, resource.TestCase{
```

Comment:
```suggestion
	resource.ParallelTest(t, resource.TestCase{
```

전체적으로 `resource.ParallelTest`로 변경 부탁드립니다.
해당 기능을 사용하면 많은 리소스들에 대해서 병렬적으로 테스트를 수행할 수 있기 때문에 테스트를 수행하는데 시간을 절약할 수 있습니다~ 😄

https://github.com/hashicorp/terraform/pull/18688

Reply:
테스트케이스 `resource.ParallelTest` 사용하도록 수정하겠습니다.